packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -75,6 +78,12 @@ contract Bridge is EssentialContract, IBridge {
         _ctx.msgHash == bytes32(PLACEHOLDER);
     }
 
+    function banAddress(address addr, bool toBan) external onlyOwner nonReentrant whenNotPaused {
```

Comment:
`whenNotPaused` needed? Seems reasonable to also be able to update these when the bridge is paused if for some reason this is related to some exploit.

Reply:
It can be removed.

Reply:
Removed